app/Svg/Parser.hs

```diff
@@ -428,10 +428,31 @@ parseTransform "" = [[1, 0, 0],
 parseTransform transform =
     parseVal parser transform
     where
-        parser = P.sepEndBy (scale <|> rotate) P.spaces >> translate
-        scale = P.string "scale(" >> double >> P.spaces >> P.option 0 double
-            >> P.char ')'
-        rotate = P.string "rotate(" >> double >> P.char ')'
+        parser = P.try scale <|> 
+                  P.try rotate <|> 
+                  P.try translate <|> 
+                  P.try matrix <|> 
+                  P.try skewX <|> 
+                  P.try skewY
+        scale = do
+            _ <- P.string "scale("
+            xScale <- double
+            _ <- P.char ',' <|> P.char ' '
+            yScale <- P.option xScale double
```

Comment:
Here, as well as in lines 449 and 451 I used `P.option` because of the optional argument, even if they aren't yet supported.

Reply:
Good investigation; if you'd like to handle optional arguments in this PR, instead of using `P.option`, a better approach is to parse all arguments as a list using [`P.sepBy`](https://hackage.haskell.org/package/parsec-3.1.17.0/docs/Text-Parsec.html). This parses a list, and then you can extract the values based on the length of the list.